acceptance-tests/src/test/resources/features/site/publicationDisplay.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Single publication display details
+
+    As a compliance officer
+    I need to ensure that only permitted details of individual publication are displayed to the end users
+    So that the system does not disclose data outside of specified contexts
```

Comment:
Suggest we make this concrete to make it clearer and change "outside of specified contexts" to "until it is officially published"

Reply:
Changed.